test/e2e/nodenetworkstate_test.go

```diff
@@ -42,14 +28,19 @@ var _ = Describe("NodeNetworkState", func() {
           stp-path-cost: 100
           stp-priority: 32
 `)
-			})
-			AfterEach(func() {
-				desiredState =
-					nmstatev1.State(`interfaces:
+
+				br1Absent = nmstatev1.State(`interfaces:
   - name: br1
     type: linux-bridge
     state: absent
 `)
+			)
+
+			BeforeEach(func() {
+				updateDesiredState(namespace, br1Up)
+			})
+			AfterEach(func() {
+				updateDesiredState(namespace, br1Absent)
```

Comment:
We have to be blocking, don't we?

Reply:
Well we have to check that the bridge is really not there, so is safe to continue.

Reply:
In this PR?

Reply:
Let's remove the bridge directly in the node so we don't use the stuff we are testing, and we test bridge deletion at different thest.